pom.xml

```diff
@@ -212,8 +213,10 @@
         <javax.ws.rs-api.version>2.0</javax.ws.rs-api.version>
         <javax.websocket-api.version>1.1</javax.websocket-api.version>
         <jaxb.version>2.3.0</jaxb.version>
-        <jaxws-ri.version>2.3.2</jaxws-ri.version>
-        <jakarta.xml.bind.version>2.3.2</jakarta.xml.bind.version>
+        <jaxws-ri.version>2.3.3</jaxws-ri.version>
+        <jakarta.xml.bind.version>2.3.3</jakarta.xml.bind.version>
```

Comment:
Why are we changing other versions? shouldn't jaxws-ri be enough?

Reply:
Thanks for review ! I have update the pom for minimal changes.
After  changing jaxws-ri version, In java8 env, knox will build failed with error:
`[WARNING]
Dependency convergence error for com.sun.activation:jakarta.activation:1.2.2 paths to dependency are:
+-org.apache.knox:gateway-provider-rewrite-step-secure-query:1.4.0
  +-org.apache.knox:gateway-server:1.4.0
    +-com.sun.xml.ws:jaxws-ri:2.3.3
      +-com.sun.xml.ws:jaxws-rt:2.3.3
        +-com.sun.activation:jakarta.activation:1.2.2
and
+-org.apache.knox:gateway-provider-rewrite-step-secure-query:1.4.0
  +-org.apache.knox:gateway-server:1.4.0
    +-com.sun.xml.ws:jaxws-ri:2.3.3
      +-com.sun.xml.ws:jaxws-rt:2.3.3
        +-com.sun.xml.messaging.saaj:saaj-impl:1.5.2
          +-com.sun.activation:jakarta.activation:1.2.2
and
+-org.apache.knox:gateway-provider-rewrite-step-secure-query:1.4.0
  +-org.apache.knox:gateway-server:1.4.0
    +-com.sun.xml.ws:jaxws-ri:2.3.3
      +-com.sun.xml.ws:jaxws-eclipselink-plugin:2.3.3
        +-com.sun.mail:jakarta.mail:1.6.5
          +-com.sun.activation:jakarta.activation:1.2.1`

And in java11 env, knox will build failed with error:
`[WARNING]
Dependency convergence error for com.sun.xml.fastinfoset:FastInfoset:1.2.18 paths to dependency are:
+-org.apache.knox:gateway-provider-security-pac4j:1.5.0-SNAPSHOT
  +-org.apache.knox:gateway-server:1.5.0-SNAPSHOT
    +-com.sun.xml.ws:jaxws-ri:2.3.3
      +-com.sun.xml.ws:jaxws-rt:2.3.3
        +-com.sun.xml.fastinfoset:FastInfoset:1.2.18
and
+-org.apache.knox:gateway-provider-security-pac4j:1.5.0-SNAPSHOT
  +-org.pac4j:pac4j-saml:3.8.3
    +-org.opensaml:opensaml-security-api:3.4.3
      +-org.apache.santuario:xmlsec:2.1.5
        +-org.glassfish.jaxb:jaxb-runtime:2.3.2
          +-com.sun.xml.fastinfoset:FastInfoset:1.2.16

[WARNING]
Dependency convergence error for org.jvnet.staxex:stax-ex:1.8.3 paths to dependency are:
+-org.apache.knox:gateway-provider-security-pac4j:1.5.0-SNAPSHOT
  +-org.apache.knox:gateway-server:1.5.0-SNAPSHOT
    +-com.sun.xml.ws:jaxws-ri:2.3.3
      +-com.sun.xml.ws:jaxws-rt:2.3.3
        +-org.jvnet.staxex:stax-ex:1.8.3
and
+-org.apache.knox:gateway-provider-security-pac4j:1.5.0-SNAPSHOT
  +-org.apache.knox:gateway-server:1.5.0-SNAPSHOT
    +-com.sun.xml.ws:jaxws-ri:2.3.3
      +-com.sun.xml.ws:jaxws-rt:2.3.3
        +-com.sun.xml.stream.buffer:streambuffer:1.5.9
          +-org.jvnet.staxex:stax-ex:1.8.3
and
+-org.apache.knox:gateway-provider-security-pac4j:1.5.0-SNAPSHOT
  +-org.apache.knox:gateway-server:1.5.0-SNAPSHOT
    +-com.sun.xml.ws:jaxws-ri:2.3.3
      +-com.sun.xml.ws:jaxws-rt:2.3.3
        +-com.sun.xml.messaging.saaj:saaj-impl:1.5.2
          +-org.jvnet.staxex:stax-ex:1.8.3
and
+-org.apache.knox:gateway-provider-security-pac4j:1.5.0-SNAPSHOT
  +-org.pac4j:pac4j-saml:3.8.3
    +-org.opensaml:opensaml-security-api:3.4.3
      +-org.apache.santuario:xmlsec:2.1.5
        +-org.glassfish.jaxb:jaxb-runtime:2.3.2
          +-org.jvnet.staxex:stax-ex:1.8.1
`

We should specify the specific version for jakarta.activation、FastInfoset and stax-ex to resolve dependencies conflict.

Reply:
Thanks for the details and tracking this down. I'll take a closer look in a few days.